ARCs/arc-0034.md

```diff
@@ -0,0 +1,45 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of a governance proposal.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The purpose of this ARC is to set the parameters for proposals submitted by the community for evaluation by xGovs prior to a general governance vote for final approval.
+ 
+The Foundation’s goal is to ensure that the xGov process is defined by, and executed by, the Algorand community.
+
+The xGov program will cover evaluation of proposals regarding grants to fund early stage projects and other community input contributions.
```

Comment:
Besides possible funding from the, as of now unfinished, Q4 2022 governance vote, how will funding be directed in the future to cover this aspect of ARC 34? How far in scope does "other community input" and "early stage projects" cover and might xGov program be used to issue grants to older projects?